Gopkg.toml

```diff
@@ -36,3 +36,7 @@
 [[constraint]]
   branch = "master"
   name = "github.com/giantswarm/backoff"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/nokia/docker-registry-client"
```

Comment:
Why don't we use `github.com/docker/docker`?

Reply:
docker/docker is a client for your local docker instance.
Sadly there is no offical API client for Docker Registries.

Reply:
What is with https://godoc.org/github.com/docker/distribution/registry/client?

Reply:
I don't see how this implements our needed functionality.